ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -280,6 +280,53 @@ def test_clear_foreign_config_fail(self, mock_validate_job_queue,
             exception.DracOperationError, drac_raid.clear_foreign_config,
             self.node, 'RAID.Integrated.1-1')
 
+    @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
+                       autospec=True)
+    def test_change_physical_disk_state(self,
+                                        mock_validate_job_queue,
+                                        mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        drac_raid.change_physical_disk_state(
+            self.node,
+            mode="JBOD",
+            controllers_to_physical_disk_ids=None)
+
+        mock_client.change_physical_disk_state.assert_called_once_with(
+            "JBOD", controllers_to_physical_disk_ids=None)
+
+    @mock.patch.object(drac_raid, 'change_physical_disk_state', spec_set=True,
+                       autospec=True)
+    @mock.patch.object(drac_raid, 'commit_config', spec_set=True,
+                       autospec=True)
+    def test__change_physical_disk_conversion(self,
```

Comment:
The method name above indicates that _change_physical_disk_conversion is to be tested

Reply:
Done.